.claude/commands/analyze.md

```diff
@@ -0,0 +1,101 @@
+---
+description: Perform a non-destructive cross-artifact consistency and quality analysis across spec.md, plan.md, and tasks.md after task generation.
```

Comment:
this sounds great. that will resolve inconsistencies like the ones I was observing with Kiro